<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="spinBoxN">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>N points:</string>
    </property>
   </widget>
   <widget class="QLabel" name="spinBoxG">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>G points</string>
    </property>
   </widget>
   <widget class="QLabel" name="spinBoxP">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>P points: </string>
    </property>
   </widget>
   <widget class="QLabel" name="spinBoxE">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Exam grade:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxN_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>161</width>
      <height>29</height>
     </rect>
    </property>
    <property name="maximum">
     <number>800</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>161</width>
      <height>29</height>
     </rect>
    </property>
    <property name="maximum">
     <number>120</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>80</y>
      <width>161</width>
      <height>29</height>
     </rect>
    </property>
    <property name="maximum">
     <number>250</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_4">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>120</y>
      <width>161</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="calculatePushButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>170</y>
      <width>171</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
